templates/orgs_ext/org_home.haml

```diff
@@ -85,6 +91,12 @@
                 [[ partner.replies.last_month ]]
               %td
                 [[ partner.replies.total ]]
+              %td
+                [[ partner.cases.opened_this_month ]]
+              %td
+                [[ partner.cases.closed_this_month ]]
+              %td
+                [[ partner.cases.total ]]
```

Comment:
I'm wondering about having these totals in the same table. Is it okay to have 6 columns? (How does it look on various sized laptop screens?) We also have to keep in mind that we are also wanting to add average response times for cases as well, but maybe we can put that in another table.